cgi-bin/DW/External/Site.pm

```diff
@@ -96,7 +130,8 @@ sub get_site {
                 map { lc $_ }
                 split( /\./, $site );
 
-    return $domaintosite{"$parts[-2].$parts[-1]"} || DW::External::Site::Unknown->accepts( \@parts ) || undef;
+    return $domaintosite{"$parts[-2].$parts[-1]"} || $domaintosite{"$parts[-1]"} || 
```

Comment:
So those trailing conditions pre-exist my touching the code, sooo I'm not entirely sure what they're for. Happy to remove them if you're confident we don't need them but I assumed they wanted leaving well enough alone because they'd been put there for a reason! :-)